### PR TITLE
Bump PR

### LINKED ISSUE #1185: Release 0.23
### Problem

0.23.0 features are not yet deployed

### Solution

manual release process

- [x] Create a release/0.23 branch off of main. Ensure github test actions run
> - [x] Tag as v0.23.0-rc1
> - [x] Confirm new docker image version is deployed

- [x] Previewnet Testing
>- [x] Deploy tagged version
>- [x] Manual testing
>- [x] Run newman tests
>- [x] Run Dapp Example Bootstrap and manual tests
>- [ ] Run acceptance tests
>- [x] Run performance tests
- [x] Testnet Testing
>- [x] Deploy tagged version
>- [x] Manual testing
>- [x] Run newman tests
>- [x] Run Dapp Example Bootstrap and manual tests
>- [ ] Run acceptance tests
>- [ ] Run performance tests
* Let bake
- [ ] Tag as v0.23.0
>- [ ] git tag
>- [ ] Confirm new docker image version is deployed
>- [ ] Write up release notes and changelist
- [ ] Mainnet Testing
>- [ ] Deploy tagged version
>- [ ] Manual testing
Any bugs or missed features found should see a new ticket opened, addressed in main and cherry-picked to release/0.20  with a new rc version tagged and docker image deployed

### Alternatives

_No response_

### PR DESCRIPTION
**Pull request by ebadiere**
Bump PR

**Related issue(s)**:

Fixes #1185 

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [ ] Tested (unit, integration, etc.)

### CODE REVIEW

**Review by Nana-EC — APPROVED** [review]

File: k6/package-lock.json
Comment (AlfredoG87) [inline k6/package-lock.json:1]: Why this file has been added?

You can see on this PR that only 6 files are expected to be modified...
https://github.com/hashgraph/hedera-json-rpc-relay/pull/1101

**Review by AlfredoG87 — CHANGES_REQUESTED** [review]
Is it necessary to add the package-lock.json from the k6 sub-project to the repo?

File: k6/package-lock.json
Comment (Nana-EC) [reply k6/package-lock.json:1]: Good catch, I didn't see this. Yeah please drop this file

Comment (ebadiere) [reply k6/package-lock.json:1]: I didn't stage that file in the original commit, but it had version information updated.  I can drop it.

**Review by AlfredoG87 — APPROVED** [review]

**Pull request merged**